.github/workflows/smoketest-node14-v4.yml

```diff
@@ -1,4 +1,4 @@
-name: Smoke Test - Node 14 on Functions V4
+name: Smoke Test - Node 18 on Functions V4
```

Comment:
It looks like the latest node.js version supported by Functions is Node 20. It also looks like Node 18 is just about to become [end of life](https://nodejs.org/en/about/previous-releases#looking-for-the-latest-release-of-a-version-branch). Should we instead move to Node 20, which will remain supported for another year?